test/export/StructureDefinitionExporter.test.ts

```diff
@@ -6199,6 +6244,33 @@ describe('StructureDefinitionExporter R4', () => {
       expect(loggerSpy.getLastMessage('error')).toMatch(/File: Weaker\.fsh.*Line: 9\D*/s);
     });
 
+    it('should apply an Assignment rule on a slice without affecting the cardinality of the slice', () => {
+      // * category[Procedure] from http://hl7.org/fhir/us/minimal/RegularObservationCodes (extensible)
+      const rootValueSet = new BindingRule('category');
+      rootValueSet.valueSet = 'http://hl7.org/fhir/us/minimal/RegularObservationCodes';
+      rootValueSet.strength = 'required';
+      const procedureValueSet = new BindingRule('category[Procedure]')
+        .withFile('Weaker.fsh')
+        .withLocation([4, 8, 4, 23]);
+      procedureValueSet.valueSet = 'http://hl7.org/fhir/us/minimal/RegularObservationCodes';
+      procedureValueSet.strength = 'extensible';
+
+      observationWithSlice.rules.push(rootValueSet, procedureValueSet);
+      doc.profiles.set(observationWithSlice.name, observationWithSlice);
+      exporter.export();
+      const sd = pkg.profiles[0];
+      const rootCategory = sd.findElement('Observation.category');
+      const regularBinding = {
+        valueSet: 'http://hl7.org/fhir/us/minimal/RegularObservationCodes',
+        strength: 'required'
+      };
+      expect(rootCategory.binding).toEqual(regularBinding);
+      expect(loggerSpy.getLastMessage('error')).toMatch(
+        /Cannot override required binding with extensible/s
+      );
+      expect(loggerSpy.getLastMessage('error')).toMatch(/File: Weaker\.fsh.*Line: 4\D*/s);
+    });
```

Comment:
This test also mentions that the cardinality of the slice is not affected, but I don't see where that is tested. Does it make sense to add an additional check for that? Or is that not necessary? I was thinking something along the lines of: 

```
const sliceElement = sd.findElement('Observation.category:Procedure');
expect(sliceElement.min).toBe(0);
expect(sliceElement.max).toBe('1');
```

Reply:
This test looks like it is a duplicate of the test up on line 6107. So, I think the wrong thing got put in here somehow.

Reply:
Ohhh good catch. I skimmed the other positive case tests, so I must have missed that one. I'm not exactly sure what the intended test should test here. Do you think it is safe to remove this test? Or did you have another idea of a case to test here instead @mint-thompson?

Reply:
Based on the test description, it sounds like this scenario is already covered by the test added on line 3966. I can't think of any other test cases that need to be added for this scenario. So, it should be safe to just remove this test.

Reply:
After discussion, we will confirm the purpose of this test with @cmoesel when he returns.

Reply:
Oops!  I intended to write this test, so I copy/pasted from another test.  But then I realized that what I wanted to test was already covered elsewhere, so I didn't need it.  But I guess I forgot to delete the copy/pasted test.  I'll remove it.